modules/upgrade/pages/upgrade-script-example.adoc

```diff
@@ -2,10 +2,35 @@
 = Sample Autoinstallation Script
 
 
-This section contains an example autoinstallation script.
-Use this script as the basis for an upgrade from SLE{nbsp}12{nbsp}SP4 to SLE{nbsp}15.
+This section contains example autoinstallation scripts.
+
+If you use these profiles, please check which variables they use.
+When the uploaded profile requires variables to be set, navigate to menu:Systems[Autoinstallation > Profiles], select the profile to edit, and navigate to the [guimenu]``Variables`` tab.
+Specify the required variables, using this format:
+
+----
+<key>=<value>
+----
+
+For all installations, the ``$redhat_management_server`` variable will be set automatically and does not need to be defined.
+
+The examples uses often the ``$distrotree`` variable, which must be defined as the distribution label which is used with this profile.
+Set the variable to the same value as you select in ``Autoinstall Tree`` in the [guimenu]``Details`` tab.
+Example:
 
-.Example: Autoinstallation Script
+----
+distrotree=sles_sap15sp1-x86_64
+----
+
+when the distribution label is `sles_sap15sp1-x86_64`.
+
+
+== Upgrading from SLES{nbsp}12{nbsp}SPn to SLES{nbsp}15{nbsp}SP1
+
+Use this script as the basis for an upgrade from SLES{nbsp}12{nbsp}SPn to SLES{nbsp}15{nbsp}SP1.
+Please check the official Product documentation to find the supported upgrade paths.
```

Comment:
```suggestion
Check the official {sles} documentation to find the supported upgrade paths.
```

Reply:
I see this more general and not limited to SLES. So maybe
```suggestion
Check the official Product documentation to find the supported upgrade paths.
```

?